utilities/project-fields-validator/Earthfile

```
@@ -0,0 +1,32 @@
+VERSION 0.8
+
+IMPORT github.com/input-output-hk/catalyst-ci/earthly/python:v3.1.7 AS python-ci
+
+check:
```

```suggestion
test:
```
I suggest making this a `test` target, so that i won't run until after all the normal checks and builds have run.
Then its only going to block publishing/release or merging.